From eee94b22508a066f7b9bc1ae05d2d85982e73959 Mon Sep 17 00:00:00 2001
From: heitbaum <heitbaum@noreply.codeberg.org>
Date: Thu, 1 May 2025 05:34:30 +0000
Subject: [PATCH] Allow build with CMake 4.0.0

CMake 4.0.0 deprecates CMake syntax < 3.10. Update to using a minimum of 3.10.

ref: https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d181300..083b5f1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1.0)
+cmake_minimum_required(VERSION 3.10)
 project(id3tag VERSION 0.16.3)
 
 option(BUILD_SHARED_LIBS "Build dynamic library" ON)
